packages/core/src/child-proxy/ChildProcessProxyWorker.ts

```diff
@@ -52,8 +52,8 @@ export default class ChildProcessProxyWorker {
     this.log = getLogger(ChildProcessProxyWorker.name);
     this.handlePromiseRejections();
     let injector = buildChildProcessInjector(message.options);
-    const locals = message.additionalInjectableValues as any;
-    for (const token of Object.keys(locals)) {
+    const locals = message.additionalInjectableValues as Record<string, unknown>;
+    for (const token in locals) {
```

Comment:
`for` ... `in` will also iterate over properties on prototype objects. But in this case, it doesn't matter, because `additionalInjectableValues`'s prototype is `Object` (and does not have any properties to iterate over).

https://stackoverflow.com/questions/43446241/for-in-or-for-of-object-keys

Reply:
Yea, I checked it first of course to see if it breaks :)